app/src/test/java/com/kickstarter/viewmodels/ProjectSearchResultHolderViewModelTest.java

```diff
@@ -0,0 +1,67 @@
+package com.kickstarter.viewmodels;
+
+import android.util.Pair;
+
+import com.kickstarter.KSRobolectricTestCase;
+import com.kickstarter.factories.PhotoFactory;
+import com.kickstarter.factories.ProjectFactory;
+import com.kickstarter.models.Photo;
+import com.kickstarter.models.Project;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import rx.observers.TestSubscriber;
+
+public final class ProjectSearchResultHolderViewModelTest extends KSRobolectricTestCase {
+  private ProjectSearchResultHolderViewModel.ViewModel vm;
+  private final TestSubscriber<String> projectImage = new TestSubscriber<>();
+  private final TestSubscriber<String> projectName = new TestSubscriber<>();
+  private final TestSubscriber<Pair<Integer, Integer>> projectStats = new TestSubscriber<>();
+  private Project project;
```

Comment:
let's create a new immutable instance of `project` in each test case rather than mutating this file global variable